src/components/dialogs/contingency-list/explicit-naming/explicit-naming-form.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import {
+    CONTINGENCY_NAME,
+    EQUIPMENT_IDS,
+    EQUIPMENT_TABLE,
+    ROW_UUID,
+} from '../../../utils/field-constants';
+import { FormattedMessage, useIntl } from 'react-intl';
+import React, { useCallback, useMemo } from 'react';
+import CustomAgGridTable, {
+    ROW_DRAGGING_SELECTION_COLUMN_DEF,
+} from '../../../utils/rhf-inputs/ag-grid-table-rhf/custom-ag-grid-table';
+import chipsArrayEditor from '../../../utils/rhf-inputs/ag-grid-table-rhf/cell-editors/chips-array-editor';
+import { gridItem } from '../../../utils/dialog-utils';
+import yup from '../../../utils/yup-config';
+import { Grid } from '@mui/material';
+import Alert from '@mui/material/Alert';
+
+export const getExplicitNamingSchema = (id) => ({
+    [id]: yup.array().of(
+        yup.object().shape({
+            [CONTINGENCY_NAME]: yup.string().nullable(),
+            [EQUIPMENT_IDS]: yup.array().of(yup.string().nullable()),
+        })
+    ),
+});
+const suppressEnter = (params) => {
+    const KEY_ENTER = 'Enter';
+    const event = params.event;
+    const key = event.key;
+    return key === KEY_ENTER;
+};
+
+const ExplicitNamingForm = () => {
+    const intl = useIntl();
+    const columnDefs = useMemo(() => {
+        return [
+            ...ROW_DRAGGING_SELECTION_COLUMN_DEF,
+            {
+                headerName: intl.formatMessage({ id: 'elementName' }),
+                field: CONTINGENCY_NAME,
+                editable: true,
+                singleClickEdit: true,
+            },
+            {
+                headerName: intl.formatMessage({ id: 'equipments' }),
+                field: EQUIPMENT_IDS,
+                suppressKeyboardEvent: (params) => {
+                    return suppressEnter(params);
+                },
+                autoHeight: true,
+                wrapText: true,
+                singleClickEdit: true,
+                cellRenderer: chipsArrayEditor,
+                cellRendererParams: {
+                    name: EQUIPMENT_TABLE,
+                },
+            },
+        ];
+    }, [intl]);
+
+    const fromCsvDataToFormValues = useCallback((results) => {
```

Comment:
results -> csvData
It was the name used before and it's more clear

Reply:
i changed to getDataFromCsvFile, do you have a better name ?

Reply:
No I mean the name of the variable "results", not the function
I think csvData was more clear, we should avoid "results" when we can.

Reply:
change it to csvData